backend/clap/src/main/java/zpi/ppea/clap/service/SubmissionService.java

```diff
@@ -40,4 +42,12 @@ public DetailsSubmissionResponseDto getDetailedSubmission(Integer submissionId)
         return dto;
     }
 
+    private void checkAccessToResource(Integer submissionId) {
+        SubmissionsResponse allAssessorsSubmissions = dataStoreBlockingStub.getSubmissions(
+                SubmissionRequest.newBuilder().setAssessorEmail(valueConfig.getFirebaseEmail()).build()
+        );
+        if (!allAssessorsSubmissions.getSubmissionsList().stream().map(Submission::getSubmissionId).toList().contains(submissionId))
```

Comment:
I don't think we need to map the list, [anyMatch](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#anyMatch-java.util.function.Predicate-) should be sufficient.